x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/policy_json_flyout.tsx

```diff
@@ -37,48 +38,56 @@ interface Props {
 /**
  * Ensure that the JSON we get from the from has phases in the correct order.
  */
-const prettifyFormJson = (policy: SerializedPolicy): SerializedPolicy => ({
-  ...policy,
-  phases: {
+const prettifyFormJson = (policy: SerializedPolicy): PolicyJson => {
+  const phasesInOrder = {
     hot: policy.phases.hot,
     warm: policy.phases.warm,
     cold: policy.phases.cold,
     frozen: policy.phases.frozen,
     delete: policy.phases.delete,
-  },
-});
+  };
+  if (policy._meta) {
+    return {
+      phases: phasesInOrder,
+      _meta: {
```

Comment:
nit: is there a specific reason to not simply do this (as we do with the phase above)

```js
return {
  phases: phasesInOrder,
  _meta: policy._meta
}
```